Source/Core/Core/HotkeyManager.cpp

```diff
@@ -36,7 +36,7 @@ const std::string hotkey_labels[] = {
     _trans("Start Recording"),
     _trans("Play Recording"),
     _trans("Export Recording"),
-    _trans("Read-only mode"),
+    _trans("Read-Only mode"),
```

Comment:
I think it is fine to capitalize like that: http://www.accu-assist.com/grammar-tips-archive/GrammarTip_capitalization-titles-headings-hyphenated-words.htm but it can probably be debated.